jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-client/src/main/java/org/jbpm/console/ng/cm/client/util/AbstractCaseInstancePresenter.java

```diff
@@ -69,9 +69,9 @@ public void findCaseInstance() {
     }
 
     protected boolean isCaseInstanceValid() {
-        return isNullOrEmpty(serverTemplateId) == false ||
-               isNullOrEmpty(containerId) == false ||
-               isNullOrEmpty(caseId)  == false;
+        return isNullOrEmpty(serverTemplateId) == false
```

Comment:
serverTemplateId is not actually in use at the moment, fixing the validation would mean that you have to exclude it from the check. Keeping only containerId and caseId